mocca-resiliency4j/src/main/java/com/paypal/mocca/client/MoccaResilience4j.java

```diff
@@ -126,5 +130,78 @@ public Builder bulkhead(final Bulkhead bulkhead) {
             this.feignBuilder = feignBuilder.withBulkhead(bulkhead);
             return this;
         }
+
+        /**
+         * Adds a Fallback to the decorator chain
+         *
+         * @param fallBack the fallback object to be set to in this builder
+         * @return this builder
+         */
+        public Builder fallback(final Object fallBack) {
+            this.feignBuilder = feignBuilder.withFallback(fallBack);
+            return this;
+        }
+
+        /**
+         * Adds a Fallback to the decorator chain using the exception filter predicate
+         *
+         * @param fallBack the fallback object to be set to in this builder
+         * @param filter Predicate for the exception filter
+         * @return this builder
+         */
+        public Builder fallback(final Object fallBack, Predicate<Exception> filter) {
+            this.feignBuilder = feignBuilder.withFallback(fallBack, filter);
+            return this;
+        }
+
+        /**
+         * Adds a Fallback to the decorator chain using the exception filter class
+         *
+         * @param fallBack the fallback object to be set to in this builder
+         * @param filter Class for the exception filter
+         * @return this builder
+         */
+        public Builder fallback(final Object fallBack, Class<? extends Exception> filter) {
+            this.feignBuilder = feignBuilder.withFallback(fallBack, filter);
+            return this;
+        }
+
+        /**
+         * Adds a Fallback factory to the decorator chain
+         *
+         * @param fallBackFactory the fallback factory function to be set in this builder
+         * @return this builder
+         */
+        public Builder fallbackFactory(Function<Exception, ?> fallBackFactory ) {
+            this.feignBuilder = feignBuilder.withFallbackFactory(fallBackFactory);
+            return this;
+        }
+
+        /**
+         * Adds a Fallback factory to the decorator chain using the exception filter class
+         *
+         * @param fallbackFactory the fallback factory function to be set in this builder
+         * @param filter Class for the exception filter
+         * @return this builder
+         */
+        public Builder fallbackFactory(Function<Exception, ?> fallbackFactory,
+                                       Class<? extends Exception> filter) {
+            this.feignBuilder = feignBuilder.withFallbackFactory(fallbackFactory, filter);
+            return this;
+        }
+
+        /**
+         * Adds a Fallback factory to the decorator chain using the exception filter predicate
+         *
+         * @param fallbackFactory the fallback factory function to be set in this builder
+         * @param filter Predicate for the exception filter
```

Comment:
`predicate` in lower case please. Always start the parameter javadoc in lower case.

Please do the same in the other methods you added.

Thanks.